activejob/lib/active_job/core.rb

```diff
@@ -20,6 +20,9 @@ module Core
 
       # ID optionally provided by adapter
       attr_accessor :provider_job_id
+
+      # I18n.locale to be used during the job.
+      attr_accessor :locale
```

Comment:
Don't you think that this is too general name? I'm currently upgrading to Rails 4.2.4 and getting errors like:
```
NoMethodError:
  private method `locale' called for #<SomeJob:0x007f1053505900>
```

In my job, I have a private method `locale` defined